airbyte-ci/connectors/pipelines/pipelines/commands/groups/connectors.py

```diff
@@ -87,45 +87,26 @@ def connectors(
 
     ctx.ensure_object(dict)
     ctx.obj["use_remote_secrets"] = use_remote_secrets
-    ctx.obj["connector_names"] = names
-    ctx.obj["connector_languages"] = languages
-    ctx.obj["release_states"] = release_stages
-    ctx.obj["modified"] = modified
+    ctx.obj["selected_connectors_names"] = set(names) if names else set()
+    ctx.obj["selected_connectors_languages"] = set(languages) if languages else set()
+    ctx.obj["selected_connectors_release_stages"] = set(release_stages) if release_stages else set()
+    ctx.obj["select_modified_connectors"] = modified
     ctx.obj["concurrency"] = concurrency
     ctx.obj["execute_timeout"] = execute_timeout
-
-    all_connectors = get_all_released_connectors()
-
-    # We get the modified connectors and downstream connector deps, and files
-    modified_connectors_and_files = get_modified_connectors(ctx.obj["modified_files"])
-
-    # We select all connectors by default
-    # and attach modified files to them
-    selected_connectors_and_files = {connector: modified_connectors_and_files.get(connector, []) for connector in all_connectors}
-
-    if modified:
-        selected_connectors_and_files = modified_connectors_and_files
-    if names:
-        selected_connectors_and_files = {
-            connector: selected_connectors_and_files[connector]
-            for connector in selected_connectors_and_files
-            if connector.technical_name in names
-        }
-    if languages:
-        selected_connectors_and_files = {
-            connector: selected_connectors_and_files[connector]
-            for connector in selected_connectors_and_files
-            if connector.language in languages
-        }
-    if release_stages:
-        selected_connectors_and_files = {
-            connector: selected_connectors_and_files[connector]
-            for connector in selected_connectors_and_files
-            if connector.release_stage in release_stages
-        }
-
-    ctx.obj["selected_connectors_and_files"] = selected_connectors_and_files
-    ctx.obj["selected_connectors_names"] = [c.technical_name for c in selected_connectors_and_files.keys()]
+    ctx.obj["all_connectors"] = get_all_released_connectors()
```

Comment:
nit: the `get_all_released_connectors()` name is a bit confusing. Reading the name and seeing `released` I would expect this to only have things that are published, but I _think_ it returns all connectors.

What does `released` refer to here?